package.json

```diff
@@ -48,50 +48,49 @@
     "dependencies": {
         "@aws-sdk/client-ses": "^3.100.0",
         "@aws-sdk/credential-provider-node": "^3.100.0",
-        "@fastify/autoload": "^5.2.0",
```

Comment:
Is autoload no longer a requirement?

Reply:
Correct - we used it early on but removed it. It was overlooked here.